src/zenml/orchestrators/step_launcher.py

```diff
@@ -286,27 +285,10 @@ def launch(self) -> None:
                 logger.info(f"Step `{self._step_name}` has started.")
 
                 try:
-                    # here pass a forced save_to_file callable to be
-                    # used as a dump function to use before starting
-                    # the external jobs in step operators
-                    if isinstance(
-                        logs_context,
-                        step_logging.PipelineLogsStorageContext,
-                    ):
-                        force_write_logs = partial(
-                            logs_context.storage.save_to_file,
-                            force=True,
-                        )
-                    else:
-
-                        def _bypass() -> None:
-                            return None
-
-                        force_write_logs = _bypass
                     self._run_step(
                         pipeline_run=pipeline_run,
                         step_run=step_run,
-                        force_write_logs=force_write_logs,
+                        force_write_logs=lambda: None,
```

Comment:
If nothing is using this, we can simply remove it from that class?

Reply:
Adding this back in for the time being. Explanation [here](https://github.com/zenml-io/zenml/pull/3855/files#r2235308173).